hbase-server/src/main/java/org/apache/hadoop/hbase/master/replication/ReplicationPeerManager.java

```diff
@@ -271,6 +271,15 @@ private void setPeerState(String peerId, boolean enabled) throws ReplicationExce
       desc.getSyncReplicationState()));
   }
 
+  public boolean getPeerState(String peerId) throws ReplicationException {
```

Comment:
Better add a method to get the ReplicationPeerDescription? Anyway, not a big problem since this class is IA.Private.